.claude/skills/ios-to-android-port/SKILL.md

```
@@ -0,0 +1,178 @@
+| `DeviceClient/` | `foundation/device/device-client/` |
+| `DeviceId/` | `foundation/device/device-id/` |
+| `DeviceProfile/` | `foundation/device/device-profile/` |
+
+Class and file names are intentionally kept in sync between platforms.
+
+## Step 1 — Fetch the iOS diff
+
+Use the GitHub API to get the diff for the feature branch or PR:
+
+**For a branch:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)